fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/data/SavingsAccountTransactionDataValidator.java

```diff
@@ -98,6 +98,9 @@ public void validate(final JsonCommand command) {
         final BigDecimal transactionAmount = this.fromApiJsonHelper.extractBigDecimalWithLocaleNamed(transactionAmountParamName, element);
         baseDataValidator.reset().parameter(transactionAmountParamName).value(transactionAmount).notNull().positiveAmount();
 
+        final String note = this.fromApiJsonHelper.extractStringNamed(noteParamName, element);
```

Comment:
what about the enums specified in ticke?

Reply:
I am closing this PR as you mention in ticket..